python/orca/dev/test/run-pytests-ray.sh

```diff
@@ -40,31 +40,26 @@ fi
 
 ray stop -f
 
-echo "Running orca learn ray backend tests"
-python -m pytest -v test/bigdl/orca/learn/ray \
-      --ignore=test/bigdl/orca/learn/ray/pytorch/test_estimator_horovod_backend.py \
-      --ignore=test/bigdl/orca/learn/ray/tf/ \
-      --ignore=test/bigdl/orca/learn/ray/ctx/ \
-      --ignore=test/bigdl/orca/learn/ray/mxnet/
+echo "Running orca learn pytorch ray backend tests"
+python -m pytest -v test/bigdl/orca/learn/ray/pytorch/
```

Comment:
it is better practice not to run each subfolder separately, still use ignore so that when adding a new folder, it won't be missed.